CMakeLists.txt

```diff
@@ -466,7 +466,16 @@ else (WITH_THREADS AND Threads_FOUND)
   set (NO_THREADS 1)
 endif (WITH_THREADS AND Threads_FOUND)
 
-set (TEST_SRC_DIR \"${CMAKE_CURRENT_SOURCE_DIR}\")
+# fopen/open on Cygwin can not handle unix-type paths like /home/.... 
+# therefore we translate TEST_SRC_DIR to windows-path.
+if (CYGWIN)
+  EXECUTE_PROCESS(COMMAND cygpath.exe -m ${CMAKE_CURRENT_SOURCE_DIR}
+                  OUTPUT_VARIABLE TEST_SRC_DIR)
+  string (STRIP ${TEST_SRC_DIR} TEST_SRC_DIR)
```

Comment:
Can be replaced by the `OUTPUT_STRIP_TRAILING_WHITESPACE` property in `execute_process`.

Reply:
done